### ISSUE TITLE
*Critical* No accessibility on Calamares make blind people unable to use Linux by themselves like before

### ISSUE THREAD

**Issue by entraide-net** [calamares/calamares#2352]
Whatever the hardware of software setup, Calamares need basic accessibility.

Expectation is simple, adoption of Calamares by Manjaro killed Sonar Linux (Jonathan Nadeau blind developper/FSF intern) and it slowly spreaded to most distributions making visual impaired users unable to use Linux by themselves like before.

This bug is unfixed for maybe 7 years.

Example of the issue :

https://www.youtube.com/watch?v=WWx07b4IVbo&t=1960s

**Comment by dalto8** [comment]
This is a duplicate of https://github.com/calamares/calamares/issues/1294

**Comment by mdyer1** [comment]
Also, if this has not been closed yet, Debian testing works with orca and the calemares installer.  Jordon OLibisy did a youtube video on this and it worked fine for me lastnight in qa vm.  It would appear that accessibility is getting there.

 

Matthew
